elements/lisk-dpos/src/delegates_list.ts

```diff
@@ -494,10 +481,17 @@ export class DelegatesList {
 		if (!forgersListBuffer) {
 			throw new Error(`No delegate list found for round: ${round.toString()}`);
 		}
-		const forgersList = decodeForgerList(forgersListBuffer);
 
-		const delegateAddresses = forgersList.find(fl => fl.round === round)
-			?.delegates;
+		const forgersListDecoded = codec.decode(
```

Comment:
```suggestion
		const forgersListDecoded = codec.decode<ForgersList>(
```
same as above

Reply:
As discussed that it fires more changes. We leave as part of fix of types in lisk-codec. Thanks for this suggestion.